src/main/java/org/sasanlabs/vulnerability/types/VulnerabilityType.java

```diff
@@ -2,15 +2,46 @@
 
 /** @author KSASAN preetkaran20@gmail.com */
 public enum VulnerabilityType {
-    XSS,
-    OPEN_REDIRECT,
-    NULL_BYTE,
-    PATH_TRAVERSAL,
-    UNRESTRICTED_FILE_UPLOAD,
-    LFI,
-    RFI,
-    COMMAND_INJECTION,
-    XXE,
-    SQL_INJECTION,
-    VULNERABLE_JWT_IMPLMENTATION
+
+    // SQL Injection
+    BLIND_SQL_INJECTION(89, 19),
+
+    ERROR_BASED_SQL_INJECTION(89, 19),
+    UNION_BASED_SQL_INJECTION(89, 19),
+
+    // XSS
+    REFLECTED_XSS(79, 8),
+    PERSISTENT_XSS(79, 8),
+    DOM_BASED_XSS(79, 8),
+
+    // JWT Related
+    CLIENT_SIDE_VULNERABLE_JWT(null, null),
+    SERVER_SIDE_VULNERABLE_JWT(null, null),
+    INSECURE_CONFIGURATION_JWT(null, null),
+
+    PATH_TRAVERSAL(22, 33),
+    COMMAND_INJECTION(77, 31),
+
+    UNRESTRICTED_FILE_UPLOAD(434, null),
+
+    OPEN_REDIRECT_3XX_STATUS_CODE(601, 38),
+
+    // XXE Vulnerability
+    XXE(611, 43);
+
+    private Integer CWE_ID;
```

Comment:
I suggest camel case, cweID and wascID and then generate the getter and setters as getCWEID(), getWASCID()